vimtutor.bat

```diff
@@ -10,7 +10,20 @@
 :: When that also fails, it uses the English version.
 
 :: Use Vim to copy the tutor, it knows the value of $VIMRUNTIME
-FOR %%d in (. "%TMP%" "%TEMP%") DO IF EXIST %%d\nul SET TUTORCOPY=%%d\$tutor$
+FOR %%d in (. %TMP% %TEMP%) DO (
+call :test_dir %0 %%d
+IF NOT ERRORLEVEL 1 GOTO dir_ok
```

Comment:
```suggestion
    call :test_dir %0 %%d
    IF NOT ERRORLEVEL 1 GOTO dir_ok
```
Indenting would be better.

Reply:
I just followed the current convention. The other if-blocks are not indented, either. Should I indent just this block?

Reply:
Other if-commands are one-liners, not blocks, so they are not indented.
I think it's better to indent this for-block.

Reply:
I see that I wrongly regarded some statements below an IF as a block—it is not, as the IF is a GOTO statement...

I'll change.